### PR TITLE
Remove Oracle To Pg scenario support

### PR DESCRIPTION
**Pull request by sumitdvlp**
We need to remove Oracle to PG scenario, part of the process is to remove the support to upload OCI driver using the extension and all modules related to Oracle to PG scenario.
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by temandr** [comment]
Don't make changes to the vendored_sdk folder - this folder contains generated file from autorest.

**Comment by temandr** [comment]
Make changes to the _help.py to update the documentations for the commands.

**Comment by temandr** [comment]
Make sure to update the version of the extension. See previous PRs on how to do that.

**Comment by temandr** [comment]
@yonzhan @jsntcy @qwordy This is ready for review. Build is failing due to this error which is unrelated to this change:
`AssertionError: Request to https://pahealy.blob.core.windows.net/azext-fzf/fzf-1.0.2-py2.py3-none-any.whl failed with 403`

**Review by temandr — APPROVED** [review]

**Comment by yonzhan** [comment]
@jsntcy to help with

**Comment by temandr** [comment]
@yonzhan @jsntcy Any updates?

**Comment by temandr** [comment]
@yonzhan @jsntcy Any updates? This has been ready for review for the past 18 days.

**Comment by jsntcy** [comment]
@sumitdvlp, @temandr, please rebase on `master` branch to fix CI failure.

**Comment by sumitdvlp** [comment]
@yonzhan @jsntcy @qwordy This is ready for review.

**Review by jsntcy — APPROVED** [review]

File: src/dms-preview/azext_dms/__init__.py [hunk 1]
```diff
@@ -3,28 +3,27 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+from azure.cli.command_modules.dms.commands import dms_api_exception_handler
```
Comment (kairu-ms) [inline src/dms-preview/azext_dms/__init__.py:6]: `dms_api_exception_handler` does not exist in azure-cli repo

**Pull request merged**